### DISCUSSION TITLE
Just wanted to say thanks, Donate?

### DISCUSSION THREAD

**Discussion by JustinGrote** [ChristopherHX/runner.server#200] [General]
Hey @ChristopherHX, just wanted to say I'm super impressed with all the improvements you've made such as artifacts support and whatnot, your solution has been the superior GHA emulator for me and has saved me countless hours of commit-run-repeat. Do you have a donate and/or charity link?

Thanks for all you do and your hard work.

**Comment by ChristopherHX** [comment]
> Do you have a donate and/or charity link?

I haven't really set up any donation infrastructure, since I'm a student and have no knowledge about that topic.
I can provide you my own paypal link [paypal.me/christopherhx
](https://paypal.me/christopherhx).
